modules/mobile/app/controllers/mobile/v0/demographics_controller.rb

```diff
@@ -11,7 +11,7 @@ class DemographicsController < ApplicationController
       def index
         response = service.get_demographics
         raise_error(response) if response.status != 200
-        render json: Mobile::V0::DemographicsSerializer.new(@current_user.icn, response)
+        render json: Mobile::V0::DemographicsSerializer.new(@current_user.uuid, response)
```

Comment:
this is the only instance of using the icn and I think icn is considered PII while uuid is not. So safer to change it to use uuid